### ISSUE TITLE
"sop armor" double armors already ASCII Armor input

### ISSUE THREAD

**Issue by guillemj** [vanitasvitae/pgpainless#335]
The latest SOP draft specifies that if the input is already ASCII Armor, the operation should be idempotent, and as such not rearmor it. But it looks like the command currently rearmors it:

```
$ cat file.asc
-----BEGIN PGP SIGNATURE-----

wr0EABYKAG8FgmNn+7oJEF8IeVKOdEodRxQAAAAAAB4AIHNhbHRAbm90YXRpb25z
LnNlcXVvaWEtcGdwLm9yZ6I+NlLnxa3+q9VIr0tkR75ehn2pOJ0a5T7TbEKamHeH
FiEEld5RAbTaqSHOdkqUXwh5Uo50Sh0AADADAP0edQHN/mcxubIU8Z8RTbdBe0uu
AyzTI/4rIjImBPRoxgEAoWB+JczVmgeVRdkm8BW77PqFailPUf0mnokZZszz+Qc=
=qfwq
-----END PGP SIGNATURE-----
$ pgpainless-cli armor <file.asc
-----BEGIN PGP MESSAGE-----
Version: PGPainless

LS0tLS1CRUdJTiBQR1AgU0lHTkFUVVJFLS0tLS0KCndyMEVBQllLQUc4RmdtTm4r
N29KRUY4SWVWS09kRW9kUnhRQUFBQUFBQjRBSUhOaGJIUkFibTkwWVhScGIyNXoK
TG5ObGNYVnZhV0V0Y0dkd0xtOXlaNkkrTmxMbnhhMytxOVZJcjB0a1I3NWVobjJw
T0owYTVUN1RiRUthbUhlSApGaUVFbGQ1UkFiVGFxU0hPZGtxVVh3aDVVbzUwU2gw
QUFEQURBUDBlZFFITi9tY3h1YklVOFo4UlRiZEJlMHV1CkF5elRJLzRySWpJbUJQ
Um94Z0VBb1dCK0pjelZtZ2VWUmRrbThCVzc3UHFGYWlsUFVmMG1ub2taWnN6eitR
Yz0KPXFmd3EKLS0tLS1FTkQgUEdQIFNJR05BVFVSRS0tLS0tCg==
=O4Eq
-----END PGP MESSAGE-----
```

I'm not entirely sure whether this belongs here or in the sop-java repo. I found this in the context of adding pgpainless-cli as an alternative SOP implementation in the upcoming OpenPGP multi-backend support for dpkg.

**Comment by vanitasvitae** [comment]
Ah, good catch! Thanks for the report, I will work on a fix!

**Comment by vanitasvitae** [comment]
This is fixed with 48005da7f3c6ecff2c3a3ad27e7414c9ab023811
